config/hsc/fgcmFitCycle.py

```diff
@@ -73,6 +73,17 @@
                             'N921': True,
                             'y': True,
                             'N1010': False}
+config.ccdGrayFocalPlaneDict = {'N387': True,
+                                'g': True,
+                                'r': True,
+                                'i': True,
+                                'N816': True,
+                                'z': True,
+                                'N921': True,
+                                'y': True,
+                                'N1010': True}
+config.ccdGrayFocalPlaneFitMinCcd = 50
```

Comment:
Some comment on this choice would be helpful...where do the tradeoffs lie with slightly fewer CCDs?